### PR TITLE
[ADD] Adding web_list_url_widget module

### PR DESCRIPTION
**Pull request by daramousk**
Use `widget="url"` on listview columns, a copy of https://github.com/odoo/odoo/commit/4caeda8669484f5a57b3737e7c35e7a14246d85f

### CODE REVIEW

File: web_widget_listview_url/README.rst [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+This module was written to extend the functionality of ListView to support URLs
+and allow you transform any text field into an hyperlink.
+
+Usage
+=====
+
+To use this module, you need to:
+
+#. go to ...
```
Comment (hbrunn) [inline web_widget_listview_url/README.rst]: give an example

**Review by hbrunn — CHANGES_REQUESTED** [review]
check travis. Also add a view modification as demo data where the widget is used so people can test on runbot.

**Comment by hbrunn** [comment]
oh, and given this is not our code, add https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#backporting-odoo-modules

**Comment by daramousk** [comment]
@hbrunn

**Comment by daramousk** [comment]
@hbrunn Anything else here?

**Review by hbrunn — APPROVED** [review]

**Comment by hbrunn** [comment]
no, it's just that pings get lost sometimes

File: web_widget_listview_url/static/src/js/web_list_url_widget.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+//Copyright 2017 Therp BV <http://therp.nl>
+//License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+openerp.web_widget_listview_url = function(instance) {
+    instance.web.list.Url = instance.web.list.Column.extend({
+        PROTOCOL_REGEX: /^(?!\w+:?\/\/)/,
+        _format: function(row_data, options) {
+            var value = row_data[this.id].value;
+            if (value) {
+                return _.template(
```
Comment (Tardo) [inline web_widget_listview_url/static/src/js/web_list_url_widget.js]: Why not use string templates here?
https://github.com/lukehoban/es6features#template-strings

**Review by andhit-r — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Following naming conventions, you should name this as `web_widget_url_list`, as it's an extension to URL widget for being shown on list views.

**Comment by pedrobaeza** [comment]
Sorry, it should be `web_widget_url_listview`, as for consistency, other modules called it that last part this way.

**Comment by daramousk** [comment]
@pedrobaeza Should be fine now.

**Pull request merged**